### ISSUE TITLE
This may be due to the fact that you're running Noesis on a unicode path

### ISSUE THREAD

**Issue by fatboyzz** [alphazolam/fmt_RE_MESH-Noesis-Plugin#13]
I try to use [blender addon](https://github.com/NSACloud/Blender-RE-Mesh-Noesis-Wrapper) which use this plugin.
The addon generate commands like following.

```
D:\Game\MonsterHunterRise_Mod\Tool\noesisv4466\Noesis.exe ?cmode "D:\Game\MonsterHunterRise_Mod\Others\mod\f\pl501\f_body501.mesh.2109148288" "C:\Users\Public\Software\blender-scripts\addons_contrib\Blender-RE-Mesh-Noesis-Wrapper-main\TempFBX\import\f_body501.fbx"
```

After entering these commands in a console I got.

```
D:\Game\MonsterHunterRise_Mod\Playground>Noesis console mode activated.
MessagePrompt:
----
The Python module could not be initialized. This may be due to the fact that you're running Noesis on a unicode path, where mbcs is the default filesystem encoding. Error code: -306
----
Noesis initialized.
Source file: D:\Game\MonsterHunterRise_Mod\Others\mod\f\pl501\f_body501.mesh.2109148288
Dest file: C:\Users\Public\Software\blender-scripts\addons_contrib\Blender-RE-Mesh-Noesis-Wrapper-main\TempFBX\import\f_body501.fbx
Options:
===
Output extension has set output file type to:
.fbx - Autodesk FBX
Detected file type: Unknown
Nothing was exported!
```

Need some help to solve the unicode problem.

**Comment by alphazolam** [comment]
yeah I dont know about this one. It's never happened for me so it must be some configuration problem on your system.

**Comment by CartmanBaba** [comment]
> I try to use [blender addon](https://github.com/NSACloud/Blender-RE-Mesh-Noesis-Wrapper) which use this plugin. The addon generate commands like following.
> 
> ```
> D:\Game\MonsterHunterRise_Mod\Tool\noesisv4466\Noesis.exe ?cmode "D:\Game\MonsterHunterRise_Mod\Others\mod\f\pl501\f_body501.mesh.2109148288" "C:\Users\Public\Software\blender-scripts\addons_contrib\Blender-RE-Mesh-Noesis-Wrapper-main\TempFBX\import\f_body501.fbx"
> ```
> 
> After entering these commands in a console I got.
> 
> ```
> D:\Game\MonsterHunterRise_Mod\Playground>Noesis console mode activated.
> MessagePrompt:
> ----
> The Python module could not be initialized. This may be due to the fact that you're running Noesis on a unicode path, where mbcs is the default filesystem encoding. Error code: -306
> ----
> Noesis initialized.
> Source file: D:\Game\MonsterHunterRise_Mod\Others\mod\f\pl501\f_body501.mesh.2109148288
> Dest file: C:\Users\Public\Software\blender-scripts\addons_contrib\Blender-RE-Mesh-Noesis-Wrapper-main\TempFBX\import\f_body501.fbx
> Options:
> ===
> Output extension has set output file type to:
> .fbx - Autodesk FBX
> Detected file type: Unknown
> Nothing was exported!
> ```
> 
> Need some help to solve the unicode problem.

I had the same problem as you, did you solve it?

**Comment by CartmanBaba** [comment]
hello，I had the same problem as you, did you solve it?

**Comment by Thessstar** [comment]
> hello，I had the same problem as you, did you solve it?

I just had the same problem , did you solve it? :(

**Comment by Thessstar** [comment]
I just had the same problem , did you solve it? :(
